sound/soc/soc-core.c

```diff
@@ -1656,9 +1657,14 @@ static void soc_check_tplg_fes(struct snd_soc_card *card)
 			dai_link->platforms->name = component->name;
```

Comment:
not for this PR, but in line 1649/1650 above `card->dai_link[i].name` should be `dai_link->name`